lib/logging/lib/logging.c

```diff
@@ -45,7 +46,7 @@ backstop(const int level, const char *const fmt, ...)
 }
 
 merr_t
-logging_init(const struct logging_params *const params)
+logging_init(const struct logging_params *const params, merr_stringify ctx_stringify)
```

Comment:
Seems like this should be   merr_stringify *ctx_stringify);   to make it clear that it's a pointer to a function.  Perhaps that's why you have to cast in other places I've seen?

Reply:
Sounds good.